plugins/sass.js

```diff
@@ -28,14 +28,14 @@ module.exports = function (options) {
     return function (data, args, callback) {
         lib.render({
             data: data.toString(),
-            success: function success(result) {
-                callback(null, result.css || result); // result.css for ^2
-            },
-            error: function error(err) {
-                err.status = 500; // for ^2
-                callback(err);
-            },
             includePaths: args.paths
+        }, function(err, result) {
+          if (result) {
```

Comment:
@thefourtheye the logic would just be inversed? You are suggesting I check for the error and if the error doesn't exist then the result must exist?